src/plugins/controls/public/react_controls/control_group/get_control_group_factory.tsx

```diff
@@ -265,10 +255,9 @@ export const getControlGroupEmbeddableFactory = (services: {
             /** Fetch the allowExpensiveQuries setting for the children to use if necessary */
             const fetchAllowExpensiveQueries = async () => {
               try {
-                const { allowExpensiveQueries } = await services.core.http.get<{
+                const { allowExpensiveQueries } = await coreServices.http.get<{
                   allowExpensiveQueries: boolean;
-                  // TODO: Rename this route as part of https://github.com/elastic/kibana/issues/174961
-                }>('/internal/controls/optionsList/getExpensiveQueriesSetting', {
+                }>('/internal/controls/getExpensiveQueriesSetting', {
```

Comment:
I renamed this route since it's no longer **specific** to the options list - the control group now does the fetching.